<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddNewTorrentDialog</class>
 <widget class="QDialog" name="AddNewTorrentDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>680</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="AddNewTorrentDialogLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea_2">
     <property name="frameShape">
      <enum>QFrame::Shape::NoFrame</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::SizeAdjustPolicy::AdjustToContents</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>882</width>
        <height>598</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="childrenCollapsible">
          <bool>false</bool>
         </property>
         <widget class="QFrame" name="torrentoptionsFrame">
          <layout class="QVBoxLayout" name="mainlayout_addui">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="managementLayout">
             <item>
              <widget class="QLabel" name="labelTorrentManagementMode">
               <property name="text">
                <string>Torrent Management Mode:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboTMM">
               <property name="toolTip">
                <string>Automatic mode means that various torrent properties(eg save path) will be decided by the associated category</string>
               </property>
               <item>
                <property name="text">
                 <string>Manual</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Automatic</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Orientation::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBoxSavePath">
             <property name="title">
              <string>Save at</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="FileSystemPathComboEdit" name="savePath" native="true"/>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBoxDownloadPath">
                <property name="title">
                 <string>Use another path for incomplete torrent</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="checked">
                 <bool>false</bool>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_4">
                 <item>
                  <widget class="FileSystemPathComboEdit" name="downloadPath" native="true"/>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="layoutRememberLastSavePath">
                <item>
                 <spacer name="horizontalSpacer_5">
                  <property name="orientation">
                   <enum>Qt::Orientation::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QCheckBox" name="checkBoxRememberLastSavePath">
                  <property name="text">
                   <string>Remember last used save path</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBoxSettings">
             <property name="title">
              <string>Torrent options</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <layout class="QHBoxLayout" name="categoryLayout">
                <item>
                 <widget class="QLabel" name="labelCategory">
                  <property name="text">
                   <string>Category:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="categoryComboBox">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="editable">
                   <bool>true</bool>
                  </property>
                  <property name="insertPolicy">
                   <enum>QComboBox::InsertPolicy::InsertAtTop</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item alignment="Qt::AlignmentFlag::AlignRight">
               <widget class="QCheckBox" name="defaultCategoryCheckbox">
                <property name="text">
                 <string>Set as default category</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="tagsLayout" stretch="0,0,0">
                <item>
                 <widget class="QLabel" name="tagsLabel">
                  <property name="text">
                   <string>Tags:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="tagsLineEdit">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                  <property name="placeholderText">
                   <string>Click [...] button to add/remove tags.</string>
                  </property>
                  <property name="clearButtonEnabled">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QToolButton" name="tagsEditButton">
                  <property name="toolTip">
                   <string>Add/remove tags</string>
                  </property>
                  <property name="text">
                   <string>...</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QGridLayout" name="gridLayout">
                <item row="0" column="0">
                 <widget class="QCheckBox" name="startTorrentCheckBox">
                  <property name="text">
                   <string>Start torrent</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <layout class="QHBoxLayout" name="stopConditionLayout">
                  <item>
                   <widget class="QLabel" name="stopConditionLabel">
                    <property name="text">
                     <string>Stop condition:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QComboBox" name="stopConditionComboBox"/>
                  </item>
                 </layout>
                </item>
                <item row="0" column="2">
                 <spacer name="horizontalSpacer_3">
                  <property name="orientation">
                   <enum>Qt::Orientation::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item row="1" column="0">
                 <widget class="QCheckBox" name="addToQueueTopCheckBox">
                  <property name="text">
                   <string>Add to top of queue</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QCheckBox" name="skipCheckingCheckBox">
                  <property name="text">
                   <string>Skip hash check</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QCheckBox" name="sequentialCheckBox">
                  <property name="text">
                   <string>Download in sequential order</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QCheckBox" name="firstLastCheckBox">
                  <property name="text">
                   <string>Download first and last pieces first</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QCheckBox" name="doNotDeleteTorrentCheckBox">
                  <property name="toolTip">
                   <string>When checked, the .torrent file will not be deleted regardless of the settings at the &quot;Download&quot; page of the Options dialog</string>
                  </property>
                  <property name="text">
                   <string>Do not delete .torrent file</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QLabel" name="contentLayoutLabel">
                  <property name="text">
                   <string>Content layout:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="contentLayoutComboBox">
                  <property name="currentIndex">
                   <number>0</number>
                  </property>
                  <item>
                   <property name="text">
                    <string>Original</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Create subfolder</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Don't create subfolder</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_4">
                  <property name="orientation">
                   <enum>Qt::Orientation::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="infoGroup">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="title">
              <string>Torrent information</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="0" column="0">
               <widget class="QLabel" name="labelSize">
                <property name="text">
                 <string>Size:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLabel" name="labelSizeData"/>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="labelDate">
                <property name="text">
                 <string>Date:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLabel" name="labelDateData"/>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="labelInfohash1">
                <property name="text">
                 <string>Info hash v1:</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLabel" name="labelInfohash1Data">
                <property name="textInteractionFlags">
                 <set>Qt::TextInteractionFlag::TextSelectableByMouse</set>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="labelInfohash2">
                <property name="text">
                 <string>Info hash v2:</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QLabel" name="labelInfohash2Data">
                <property name="textInteractionFlags">
                 <set>Qt::TextInteractionFlag::TextSelectableByMouse</set>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="labelComment">
                <property name="text">
                 <string>Comment:</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="QScrollArea" name="scrollArea">
                <property name="frameShape">
                 <enum>QFrame::Shape::NoFrame</enum>
                </property>
                <property name="sizeAdjustPolicy">
                 <enum>QAbstractScrollArea::SizeAdjustPolicy::AdjustToContents</enum>
                </property>
                <property name="widgetResizable">
                 <bool>true</bool>
                </property>
                <widget class="QWidget" name="scrollAreaWidgetContents">
                 <property name="geometry">
                  <rect>
                   <x>0</x>
                   <y>0</y>
                   <width>411</width>
                   <height>70</height>
                  </rect>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_2">
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QLabel" name="labelCommentData">
                    <property name="textFormat">
                     <enum>Qt::TextFormat::RichText</enum>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
                    </property>
                    <property name="wordWrap">
                     <bool>true</bool>
                    </property>
                    <property name="openExternalLinks">
                     <bool>true</bool>
                    </property>
                    <property name="textInteractionFlags">
                     <set>Qt::TextInteractionFlag::TextSelectableByKeyboard|Qt::TextInteractionFlag::TextSelectableByMouse</set>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QHBoxLayout" name="contentFilterLayout">
             <item>
              <widget class="QPushButton" name="buttonSelectAll">
               <property name="text">
                <string>Select All</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="buttonSelectNone">
               <property name="text">
                <string>Select None</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_6">
               <property name="orientation">
                <enum>Qt::Orientation::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="TorrentContentWidget" name="contentTreeView">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>1</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="contextMenuPolicy">
              <enum>Qt::ContextMenuPolicy::CustomContextMenu</enum>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::SelectionMode::ExtendedSelection</enum>
             </property>
             <property name="sortingEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonsHLayout">
     <item>
      <widget class="QCheckBox" name="checkBoxNeverShow">
       <property name="text">
        <string>Never show again</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QHBoxLayout" name="metadataLayout">
       <item>
        <widget class="QProgressBar" name="progMetaLoading">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximum">
          <number>0</number>
         </property>
         <property name="value">
          <number>-1</number>
         </property>
         <property name="textVisible">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblMetaLoading">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonSave">
         <property name="text">
          <string>Save as .torrent file...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TorrentContentWidget</class>
   <extends>QTreeView</extends>
   <header>gui/torrentcontentwidget.h</header>
  </customwidget>
  <customwidget>
   <class>FileSystemPathComboEdit</class>
   <extends>QWidget</extends>
   <header>gui/fspathedit.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
